pkgs/development/compilers/llvm/common/default.nix

```
@@ -589,7 +593,10 @@ let
             ];
             stripLen = 1;
             hash = "sha256-1NKej08R9SPlbDY/5b0OKUsHjX07i9brR84yXiPwi7E=";
-          });
+          })
+          ++ lib.optionals (lib.versions.major metadata.release_version == "19") [
```

```suggestion
          ++ lib.optionals (lib.versionOlder metadata.release_version "20") [
```

patch can apply to all clangs under 20

---

> Unless you're certain it doesn't break those builds, I would rather not include that change in this PR; that really seems something which should be done and tested separately.

the patch is currently applied unconditionally so it will be fine. additionally, can move this patch to just below the unconditional section and then will not cause the mass rebuilds.

---

Historically, LLVM git hasn't caused mass rebuilds so I'd like to continue that.

---

My bad; I had become confused and thought this patch only applied to LLVM 19 - probably got my wires crossed with the compiler-rt patch. My intent was exactly to not incur a mass rebuild. Now fixed as suggested.